### PR TITLE
correctly set archs in assorted templates

### PR DESCRIPTION
**Pull request by q66**

### CODE REVIEW

**Comment by Piraty** [comment]
vboot-utils still marks aarch64 as broken despite your addition in `archs=`

**Comment by q66** [comment]
That is ok, the archs line is correct - breakage != supported architectures

**Comment by q66** [comment]
rebase

File: srcpkgs/android-tools/template [hunk 1]
```diff
@@ -5,6 +5,7 @@ pkgname=android-tools
 #         curl -L http://git.io/vvC0Z | sh -s 5.0.2_r1 5.1.0_r1
 version=9.0.0r35
 revision=1
+archs="x86_64* i686*"
```
Comment (Johnnynator) [inline srcpkgs/android-tools/template]: Sure about this one? E.g. Alpine lists `x86 x86_64 aarch64 armv7 armhf`

Comment (Johnnynator) [reply srcpkgs/android-tools/template]: Edit: https://github.com/alpinelinux/aports/commit/67283cdbf6ef0ce2912bd28297714d6bb3976a38
Leave it as you done for now, my statement for nocross also applies for native arm, so we would need to set `-Wno-attributes` can test it somewhen later.

**Pull request merged**